frontend/src/i18n/en/translation.json

```
@@ -544,7 +545,7 @@
     "total": "Total",
     "general": "General",
     "other": "Other",
-    "no_data":" No data"
+    "no_data": " No data"
```

```suggestion
    "no_data": "No data"
```
Hi @kartik1112, can you please remove the extra space in the beginning of this value.